docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -311,47 +323,56 @@ For more information on the relationship between the elements, see the [element
-|-----------------------|--------------------|
+|-------------------------|----------------------|
 | `<label>`               | 1 or more (optional) |
-| `<icon>`                |                      |
-| `<app-control>`         |                      |
-| `<metadata>`            |                      |
-| `<datacontrol>`         |                      |
-| `<background-category>` |                      |
+| `<icon>`                | 1 or more (optional) |
+| `<app-control>`         | 1 or more (optional) |
+| `<metadata>`            | 1 or more (optional) |
+| `<datacontrol>`         | 1 or more (optional) |
+| `<background-category>` | 1 or more (optional) |
 
 **Attributes:**
 
-- `appid`  
-Application unique ID (string)  
-This can be used for launching or terminating the application explicitly.
+- `appid`
+
+  Application unique ID (string)
+
+  This can be used for launching or terminating the application explicitly.
 
-- `auto-restart`  
-Indicates whether the application is relaunched automatically if it is terminated abnormally (available values: `true`, `false`)  
-If the value is not defined, `false` is used.  
+- `auto-restart`
+
+  Indicates whether the application is relaunched automatically if it is terminated abnormally (available values: `true`, `false`)
+
+  If the value is not defined, `false` is used.
 
  > **Note**  
  > This attribute is not supported on Tizen wearable devices. Since Tizen 2.4, this attribute is not supported on all Tizen devices.
 
-- `exec`  
-Application executable file path (string)
+- `exec`
+  Application executable file path (string)
+
+- `multiple`
+
+  Indicates whether the application can be launched as a multiple (available values: `true`, `false`)
 
-- `multiple`  
-Indicates whether the application can be launched as a multiple (available values: `true`, `false`)
   > **Note**  
   > This attribute is read-only. Do not attempt to modify it.
 
-- `on-boot`  
-Indicates whether the application is launched automatically on device boot or application installation (available values: `true`, `false`)  
-If the value is not defined, `false` is used.  
+- `on-boot`
 
- > **Note**  
- > This attribute is not supported on Tizen wearable devices. Since Tizen 2.4, this attribute is not supported on all Tizen devices.
+  Indicates whether the application is launched automatically on device boot or application installation (available values: `true`, `false`)
+
+  If the value is not defined, `false` is used.
+
+  > **Note**  
+  > This attribute is not supported on Tizen wearable devices. Since Tizen 2.4, this attribute is not supported on all Tizen devices.
+
+- `taskmanage`
 
-- `taskmanage`  
-Indicates whether the application is shown in the task manager (available values: `true`, `false`)
+  Indicates whether the application is shown in the task manager (available values: `true`, `false`)
 
-- `type`  
-Tizen application type (available values: `capp`, `c++app`, `webapp`)
+- `type`
```

Comment:
Question : One empty space is neede here. Is it intended to form it in one line?

Reply:
done